playgrounds/react/src/examples/themes/dynamic-theme-example.tsx

```diff
@@ -0,0 +1,27 @@
+import { RadioButton, RadioGroup } from '@devextreme/react';
+
+import '@devextreme/styles/lib/themes/dark-scoped.css';
+
+const OPTIONS = ['light', 'dark'];
+const handleChange = async (newValue?: string) => {
+  // In reaal demo we can replace static import to fetch and link injection:
+  // const cssText = await fetch('@devextreme/styles/lib/themes/dark-scoped.css');
+  // const link = document.getElementById('theme-style') || document.createElement('style');
+  // style.innerText = cssText;
+  document.getElementById('dynamic-theme')!.className = `dx-theme-${newValue}`;
```

Comment:
Do we need direct DOM manipulation? What if we save chosen theme to the state in handleChange and set className directly in div on render?
<div id="dynamic-theme" className=`dx-theme-${chosenTheme}`>

Reply:
Currently, it's a proof of concept. I suppose in the future it can be a multi-platform helper method.